src/main.rs

```diff
@@ -118,15 +118,15 @@ fn main() {
         return None
     }
 
-    for e in event::events(&window) {
+    for e in event::events(window.clone()) {
         use event::{ ReleaseEvent, UpdateEvent, PressEvent, RenderEvent };
 
         if let Some(args) = e.update_args() {
             vm.step(args.dt as f32);
             if vm.beeping() {
-                (*window.borrow_mut()).window.set_title(BEEP_TITLE).unwrap();
+                (window.borrow_mut()).window.set_title(BEEP_TITLE);
```

Comment:
`Rustc` will complain about the unused result, see https://travis-ci.org/chip8-rust/chip8-ui/builds/57169190#L2728-L2733